takzero/src/search/node/mod.rs

```diff
@@ -61,6 +80,12 @@ impl<E: Environment> Node<E> {
         self.children.is_empty()
     }
 
+    /// Returns the Principal Variation of the search tree
+    #[must_use]
+    pub fn pv<'a>(&'a self) -> impl Iterator<Item = E::Action> + 'a {
```

Comment:
```suggestion
    pub fn principal_variation<'a>(&'a self) -> impl Iterator<Item = E::Action> + 'a {
```
I prefer to be verbose.

Reply:
Yes